.changelog/11027.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:bug
+dns: Remove dns_config.udp_answer_limit agent config option, which was had side effects (see GH-10694) when unspecified despite being deprecated and replaced by dns_config.a_record_limit in v1.0.7.

```

Comment:
should this be `config:` instead of `dns:`?

Reply:
I think `dns` is good. It is related to config as well, but I think `dns` is more specific